mycore-base/src/main/java/org/mycore/common/xml/MCRURIResolver.java

```diff
@@ -34,6 +34,7 @@
 import java.time.Instant;
```

Comment:
One more thing, ... all the changes in this class now seem to be formatting/indentation changes and an `import` that now seem unnecessary. I think, all those changes can be dropped and this class be removed from the PR.

Reply:
Good point! We have already formatted the main branch to prepare the release, so changes in 2022.06 should be as small as possible.

Reply:
done